docker-base/Dockerfile.debian

```diff
@@ -0,0 +1,77 @@
+# This Dockerfile builds our base image with gosu, dumb-init and the atlantis
```

Comment:
It would be good to move the common items between the debian and alpine bases into a separate script to DRY it up.